### ISSUE TITLE
Staging Review finding: Heading structure doesn't match content

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105698]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Education Data Migration
**Product:** WEAMS Public
**Feature:** WEAMS Public

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On a [details page for a national exam](https://staging.va.gov/education/gi-bill-comparison-tool/national-exams/4@8?examName=MAINTENANCE%20AWARENESS), the H1 heading is followed immediately by an H3 heading, with no H2 heading between them.

The page follows largely the same structure as the license, ertification, prep course details page, but those pages have an additional H2 (eg. "License") that resolves the heading structure issue.

## Recommended action

Either "Admin info" and "Test info" should be H2s, or there should be a "National exam" H2 placed after the H1 to match the structure of the other pages.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by skulka07** [comment]
Heading structure "Admin info" and "Test info" has been modified to follow standard to H2s

![Image](https://github.com/user-attachments/assets/ea410f0a-0272-45fb-9a7c-1402f68af2b4)
